src/DrupalCI/Configuration/EnvironmentVariables.php

```diff
@@ -0,0 +1,381 @@
+   *
+   * @var \Pimple\Container
+   */
+  protected $container;
+
+  /**
+   * @Variable(
+   *   environment  = "DCI_DBType",
+   *   description  = "The database type (mysql, pgsql, etc) for a given build.",
+   *   defaultValue = "mysql-5.5"
+   * )
+   */
+  public $db_type;
+
+  /**
+   * @Variable(
+   *   environment  = "DCI_DBVersion",
+   *   description  = "The version of the database that will be used.",
+   *   defaultValue = "5.5"
+   *   parser       = "parseDBVersion"
+   * )
+   */
+  public $db_version;
+
+  /**
+   * @Variable(
+   *   environment  = "DCI_DBUser",
+   *   description  = "The default database user to be used on the site database",
+   *   defaultValue = "drupaltestbot"
+   * )
+   */
+  public $dbuser;
+
+  /**
+   * @Variable(
+   *   environment  = "DCI_DBPassword",
+   *   description  = "The default database password to be used on the site",
+   *   defaultValue = "drupaltestbotpw"
+   * )
+   */
+  public $dbpassword;
+
+  /**
+   * @Variable(
+   *   environment  = "DCI_RunScript",
+   *   description  = "The default run script to be executed on the container.",
+   *   defaultValue = "/var/www/html/core/scripts/run-tests.sh"
+   * )
+   */
+  public $runscript;
+
+  /**
+   * @Variable(
+   *   environment  = "DCI_PHPInterpreter",
+   *   description  = "The php interpreter to be passed to the Run Script in the
+   *   --php argument.",
+   *   defaultValue = "/opt/phpenv/shims/php"
+   * )
+   */
+  public $php;
+
+  /**
+   * @Variable(
+   *   environment  = "DCI_Concurrency",
+   *   description  = "The value to pass to the --concurrency argument of the run script.",
+   *   defaultValue = "4"
+   * )
+   */
+  public $concurrency;
+
+  /**
+   * @Variable(
+   *   environment = "DCI_RTTypes",
+   *   description = "Types that will be used during the execution of the
+   *   runscript."
+   * )
+   *
+   * @todo Review & update
+   */
+  public $types;
+
+  /**
+   * @Variable(
+   *   environment = "DCI_RTSqlite",
+   *   description = "The sqlite file that will be used during the execution of
+   *   the run tests script."
+   * )
+   *
+   * @todo Review & update
+   */
+  public $sqlite;
+
+  /**
+   * @Variable(
+   *   environment = "DCI_RTUrl",
+   *   description = "The URL that will be used during the execution of the run
+   *   tests script."
+   * )
+   *
+   * @todo WARNING: We have been defining this before as:
+   * if (isset($_ENV['DCI_RTUrl'])) {
+   *   $this->configuration['types'] = $_ENV['DCI_RTUrl'];
+   * }
+   * Was this on purpose or was this a possible bug?
+   */
+  public $url;
+
+  /**
+   * @Variable(
+   *   environment = "DCI_RTColor",
+   *   description = "(boolean) Indicates if the output of the run tests script
+   *   should be colorized."
+   * )
+   *
+   * @todo Review & update
+   */
+  public $color;
+
+  /**
+   * @Variable(
+   *   environment      = "DCI_RTTestGroups",
+   *   environmentAlias = {"DCI_TestItem", "DCI_TestGroups"},
```

Comment:
I really like the idea of environmentAlias here, except below I think you are using it differently in the repository section.  These three truly all can be treated as aliases, below, (215-220) all of those variables are different and get *combined* to make the repositories configuration value.